src/components/GameDetail/index.jsx

```diff
@@ -47,71 +47,52 @@ export default function GameDetail() {
   }
   return (
     <div className="game-detail">
-      <h1 className="game-title">{name || <Skeleton width="400px" />}</h1>
-      <div className="game-detail-container">
-        <div className="game-detail-left">
-          {!loading ? (
-            <img className="game-image" src={background_image} alt="game" />
-          ) : (
-            <Skeleton height="400px" width="53vw" />
-          )}
-          <div className="game-description">
-            <h2>About this Game</h2>
-            <div>
-              {!loading ? (
-                description ? (
-                  parse(`${description}`)
-                ) : (
-                  `No Data`
-                )
-              ) : (
-                <Skeleton height="200px" />
-              )}
+      {!loading ? (
+        <>
+          <h1 className="game-title">{name || <Skeleton width="400px" />}</h1>
```

Comment:
write this logic in reverse order, small condition should come first so is easier to read.